Readme.md

```diff
@@ -220,6 +220,30 @@ The `method` is optional. The default value is `link`, while the possible values
 
 When using the `copy` method, the import will take longer.
 
+### Importing and export single collections
+
+_**note:**_ Since directories might have been created by the docker daemon instead of a user,
+root privileges might be necessary.
+
+#### Exporting
+Liquid has to be deployed in order to export collections. When deployed, run
+```
+./liquid exportcollection <collection_name>
+```
+The exported collection data is stored in `<collections_directory>/exported/<collection_name`
+
+#### Importing
+you need paths to the following data in order to import a collection:
+- database: the folder must contain a subfolder called data, containing the database
+- blobs: the folder contains the blobs
+- index: a `.tgz` archive containing the index
+run the following command:
+```
+./liquid importcollection <collection_name> <database_path> <blobs_path> <index_path> [method]
```

Comment:
Could we simplify this to `importcollection <name> <path> [<method>]` and expect the data at hard-coded directory names under `<path>`, having the same structure as the `export` result?

The idea would be to have these two commands just work, one after the other:
- `./liquid exportcollection testdata`
- `./liquid importcollection testdata-clone collections/exported/testdata`

Reply:
Yeah I agree, like this it's clunky. It was to be replaced by the execution of my masterplan to archive it all in one password protected file, but since that is scrapped, I should change that.